gyp/sources.txt

```diff
@@ -9,4 +11,4 @@
 <(src_loc)/spellcheck/spellcheck_utils.h
 <(src_loc)/spellcheck/spellcheck_types.h
 <(src_loc)/spellcheck/spelling_highlighter.cpp
-<(src_loc)/spellcheck/spelling_highlighter.h
\ No newline at end of file
+<(src_loc)/spellcheck/spelling_highlighter.h
```

Comment:
I'm not sure that such changes to the last lines are relevant to this commit. I think it would be better to remove these changes.

Reply:
My text editor inserted an extra EOL here that I haven't notice before.